downstream/modules/automation-hub/proc-set-community-remote.adoc

```diff
@@ -1,22 +1,22 @@
 // Module included in the following assemblies:
 // obtaining-token/master.adoc
 [id="proc-set-community-remote"]
-= Configuring the community remote repository and syncing Ansible Galaxy collections
+= Configuring the community remote repository and syncing {Galaxy} collections
 
-You can edit the *community* remote repository to sync chosen collections from Ansible Galaxy to your local {HubName}.
+You can edit the *community* remote repository to sync chosen collections from {Galaxy} to your local {HubName}.
 By default, your local {HubName} `community` repository directs to `https://galaxy.ansible.com/api/`.
 
 .Prerequisites
 
 * You have *Modify Ansible repo content* permissions.
 See https://access.redhat.com/documentation/en-us/red_hat_ansible_automation_platform/{PlatformVers}/html/managing_user_access_in_private_automation_hub/index[Managing user access in Automation Hub] for more information on permissions.
-* You have a `requirements.yml` file that identifies those collections to sync from Ansible Galaxy.
+* You have a `requirements.yml` file that identifies those collections to sync from {Galaxy}.
 See example below.
 
 .Requirements.yml example
 -----
 collections:
-  # Install a collection from Ansible Galaxy.
+  # Install a collection from {Galaxy}.
```

Comment:
The `{Galaxy}` attribute doesn't work within a codeblock.
```suggestion
# Install a collection from Ansible Galaxy.
```

Reply:
attributes don't seem to work in the master.adoc title either

Reply:
They look OK after a rebase against `main`